### ISSUE TITLE
MSVC+CUDA Test build error

### ISSUE THREAD

**Issue by crtrott** [kokkos/kokkos#5888]
In TestCuda_Spaces.cpp we see an error for CTAD in line 377:

```c++
InitFunctor iota(view);

// change to
InitFunctor<View> iota(view);
```

**Comment by PhilMiller** [comment]
Should the fix for this get picked in 4.0.1? The broken code is in 4.0.0

**Comment by masterleinad** [comment]
Fixed in #5905.
